tests/app/common.py

```diff
@@ -48,13 +48,25 @@
 }
 
 # edit to increase positive count by 1
+# remove inIcuCurrently
 NY_YESTERDAY_EDITED = {
     "state": "NY",
     "lastUpdateIsoUtc": NOW.isoformat(),
     "dateChecked": NOW.isoformat(),
     "date": YESTERDAY,
     "positive": 16,
-    "negative": 4
+    "negative": 4,
+    "inIcuCurrently": None
```

Comment:
which test(s) does blanking out `inIcuCurrently` affect and are they affected the way we'd expect?